### ISSUE TITLE
[Bug]: Ens not populating avatar

### ISSUE THREAD

**Issue by sherodtaylor** [blocknative/web3-onboard#857]
### Current Behavior

- ENS object on `Accounts` is not populating avatar for  Mainnet Ethereum 
- ENS object on `Accounts` is missing all fields on Rinkbey chain

### Onboard Version

2.0.2

### Node Version

v16.13.2

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
Mainnet

{
    "name": "sherodbuilds.eth",
    "contentHash": null
}
```
Rinkbey ens empty
```
null
```
```


### Anything else?

_No response_

**Comment by aaronbarnardsound** [comment]
Ah yes I can see that we have missed `avatar` on the ENS object, I will get that fixed in the next release very soon. 

For the Rinkeby ENS, does it show up after a few moments? The state gets set to null initially and then gets updated once the network requests resolve.

**Comment by aaronbarnardsound** [comment]
Closed via #862

**Comment by sherodtaylor** [comment]
Thank you!
